src/main/java/com/trashit/back/entity/User.java

```diff
@@ -17,6 +18,7 @@
     private String lastname;
     private String surname;
     @Column(nullable = false)
+    @JsonIgnore()
```

Comment:
A quoi ça sert ? 
Tu as surement pas besoin des parantèses par contre
```suggestion
    @JsonIgnore
```

Reply:
ça sert à gérer les formats de retour par l'api. En gros là quand on retournera le user via l'API y aura pas son mot de passe crypter! C'est mieux mdr

Reply:
Ok si c'est mieux alors x)